### PR TITLE
Updated example instructions

### PR DESCRIPTION
**Pull request by rpetrusha**
## Updated example instructions

Fixes dotnet/docs#7938

### CODE REVIEW

File: xml/System.Threading.Tasks/TaskScheduler.xml [hunk 1]
```diff
@@ -93,11 +93,11 @@
   
  To create the example, create a WPF application project in Visual Studio and assign it a name of your choice. Then do the following:  
   
-1.  In design view, drag an <xref:System.Windows.Controls.Image> control from the **Toolbox** to the design surface. In XAML view, specify the horizontal alignment as "Left." The size does not matter because the control is be dynamically resized at run time. Accept the default name, "image".  
+1.  In design view, drag an <xref:System.Windows.Controls.Image> control from the **Toolbox** to the design surface. In XAML view, specify the horizontal alignment as "Left." The size does not matter because the control is be dynamically resized at run time. In the Name textbox of the Properties window, name the control "image".
```
Comment (mairaw) [inline xml/System.Threading.Tasks/TaskScheduler.xml]: default horizontal alignment is already left, remove that?

also from what I tested, the control should be dropped on the upper left corner, otherwise results look funny.

File: xml/System.Threading.Tasks/TaskScheduler.xml [hunk 2]
```diff
@@ -93,11 +93,11 @@
   
  To create the example, create a WPF application project in Visual Studio and assign it a name of your choice. Then do the following:
```
Comment (mairaw) [inline xml/System.Threading.Tasks/TaskScheduler.xml]: Name of your choice is not really right here given the code above is assuming a namespace so if you copy and paste this code it won't work if you give a random name. So maybe we should tell folks to create a project with a specific name?

File: xml/System.Threading.Tasks/TaskScheduler.xml [hunk 3]
```diff
@@ -93,11 +93,11 @@
...
   
-2.  Drag a <xref:System.Windows.Controls.Button> control from the **Toolbox** to the lower left part of the application window. Double-click the button to add a <xref:System.Windows.Controls.Primitives.ButtonBase.Click> event handler. In XAML view, specify the <xref:System.Windows.Controls.ContentControl.Content%2A> property of the button as "Make a Mosaic" and specify its horizontal alignment as "Left". Accept the default name, "button".  
+2.  Drag a <xref:System.Windows.Controls.Button> control from the **Toolbox** to the lower left part of the application window. Double-click the button to add a <xref:System.Windows.Controls.Primitives.ButtonBase.Click> event handler. In XAML view, specify the <xref:System.Windows.Controls.ContentControl.Content%2A> property of the button as "Make a Mosaic" and specify its horizontal alignment as "Left". In the Name textbox of the Properties window, name the control "button".
```
Comment (mairaw) [inline xml/System.Threading.Tasks/TaskScheduler.xml]: same here for horizontal alignment. Default is left.

nit: Make a mosaic

In this case, you should also specify the size otherwise the default size cuts the text.

One last thing, after step 3, where you add the code to the project, you also need to connect the button Click event to the button_Click method.

**Comment by rpetrusha** [comment]
I've addressed your feedback, @mairaw. Thanks for testing the example so thoroughly!
